hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFileReader.java

```diff
@@ -98,16 +101,13 @@ public HoodieLogFileReader(FileSystem fs, HoodieLogFile logFile, Schema readerSc
     this.reverseReader = reverseReader;
     this.enableInlineReading = enableInlineReading;
     this.keyField = keyField;
+    this.keyDeDuplication = false;
```

Comment:
can we fix the naming here? we are just not storing the `key ` again in the payload right? Can we refer to this as `excludeKeyInPayload` or sth like that?

Reply:
sure, using the suggested variable naming across the code now.